### ISSUE TITLE
Api to get the encoded buyer email address

### ISSUE THREAD

**Issue by andreinacuellor** [allegro/allegro-api#9868]
Hi!

Is there a request on the Allegro API that I can get the encoded buyer email using the clintId or offer / order id? thank you

**Comment by Lukasz-Zurek** [comment]
If you want to retrieve buyer data (with encoded buyer email, address, etc.) for s specific order, you can use [GET /order/checkout-forms/{id}](https://developer.allegro.pl/documentation/#operation/getOrdersDetailsUsingGET). You can find more information [here](https://developer.allegro.pl/tutorials/process-orders-PgPMlWDr8Cv#order-details).

**Comment by andreinacuellor** [comment]
And if I have the thread id from the message center can i get also the encoded buyer email?

**Comment by Lukasz-Zurek** [comment]
No. It is not possible. [Here](https://developer.allegro.pl/tutorials/jak-zarzadzac-centrum-wiadomosci-XxWm2K890Fk#szczegolowe-informacje-o-danym-watku) you can find more information about message center resources.

**Comment by andreinacuellor** [comment]
So, the only way to get the encoded buyer email is using the request https://developer.allegro.pl/documentation/#operation/getOrdersDetailsUsingGET ?

**Comment by Lukasz-Zurek** [comment]
Yes, and also using [GET /order/checkout-forms](https://developer.allegro.pl/documentation/#operation/getListOfOrdersUsingGET).

**Comment by andreinacuellor** [comment]
Ok thanks
